IPython/terminal/interactiveshell.py

```diff
@@ -286,6 +286,16 @@ def _make_style_from_name_or_cls(self, name_or_cls):
                     Token.OutPrompt: '#990000',
                     Token.OutPromptNum: '#ff0000 bold',
                 })
+
+                # Hack: Due to limited color support on the Windows console
+                # the prompt colors will be wrong without this
+                if os.name == 'nt':
+                    style_overrides.update({
+                        Token.Prompt: '#ansidarkgreen',
```

Comment:
> It's actually prompt-toolkit that handles this. They are declared in prompt_toolkit/styles/base.py:39-48 and referenced throughout prompt-toolkit.

Yes, though be careful, we may have custom code that special case `OutPromptNum` . Though I believe it will not be affected as it it generates `OutPrompt` Tokens and not consume the. I would have to check.